app/views/external_activities/_form.html.haml

```diff
@@ -31,9 +31,10 @@
       = f.text_area :keywords, :cols => 80, :rows => 3, :class => 'mceNoEditor'
     = field_set_tag t('authoring.material_type_label') do
       = f.select :material_type,
-        [ [ Activity.display_name,      Activity.name       ],
-          [ Investigation.display_name, Investigation.name  ],
-          [ Interactive.display_name,   Interactive.name    ] ]
+        [ [ Activity.display_name,       Activity.name       ],
+          [ Investigation.display_name,  Investigation.name  ],
+          [ 'Collection', 'Collection' ],
```

Comment:
you should use `t(:collection).titleize` instead of the first `'Collection'`. That way if we translate collection someone editing the page will see the translated selection in the menu.